x-pack/plugins/security/server/lib/authorization/privileges.test.ts

```diff
@@ -21,13 +21,13 @@ test(`builds privileges correctly`, () => {
       name: 'Foo Feature',
```

Comment:
This singular test is becoming rather overwhelming to determine what we're testing where. We have some `catalogue`/`management` entries in the various features to ensure that they automatically cascade down to the privileges, but we're missing these for the `app` features/privileges, and these are only ever specified at the feature level.